### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by muthuswamyopn**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+1. Step 1
+2. Step 2
+...
+
+## Logs/Error Messages
+
+_Attach or paste appropriate logs and error messages to help us debug better._
+
+## Development Environment and Version
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: @muthuswamyopn I'd like to combine all the version requests into one section.

Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Modified @danfowler

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+_1. Step 1_
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: When rendered, this ("1. Step 1...") looks redundant...

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+_2. Step 2_
+
+...
+
+## Logs/Error Messages
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Inconsistent capitalization. Please continue with the sentence case employed in headers above.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+
+_* Development Environment (For example, XCode version 12.0)._
+
+_* SDK (For example, Omise-iOS version 4.25)._
+
+_* Package Manager (For example, Swift version 5.0)._
+
+_* Operating System (For example, Swift version 5.0)._
+
+_* Browser, if it is a web issue (For example, Swift version 5.0)._
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: This is not an example of a browser.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+
+_Specify the name and version of the following components:_
+
+_* Development Environment (For example, XCode version 12.0)._
+
+_* SDK (For example, Omise-iOS version 4.25)._
+
+_* Package Manager (For example, Swift version 5.0)._
+
+_* Operating System (For example, Swift version 5.0)._
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: This is not an example of an operating system.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+
+## Name and Version Information
+
+_Specify the name and version of the following components:_
+
+_* Development Environment (For example, XCode version 12.0)._
+
+_* SDK (For example, Omise-iOS version 4.25)._
+
+_* Package Manager (For example, Swift version 5.0)._
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: This is not an example of a package manager.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 7]
```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report
+about: Report a bug
+title: ''
+labels: ''
+assignees: ''
```
Comment (danfowler) [inline .github/ISSUE_TEMPLATE/bug-report.md]: Can we assign `@omise/maintainers`?

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Removed the steps.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 3 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Rectified.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 4 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Rectified.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 5 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Rectified.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 6 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Rectified.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 7 above)
Comment (muthuswamyopn) [reply .github/ISSUE_TEMPLATE/bug-report.md]: Assigned to `@omise/maintainers`.

**Pull request closed without merging**